cmd/podman/restore.go

```diff
@@ -40,16 +37,18 @@ func init() {
 	restoreCommand.SetHelpTemplate(HelpTemplate())
 	restoreCommand.SetUsageTemplate(UsageTemplate())
 	flags := restoreCommand.Flags()
+	// TODO: add ContainerStateCheckpointed
```

Comment:
I don't know if this makes sense, necessarily - you can restart the container after a checkpoint is made, no? It seems more like we should have a function on the container `HasCheckpoint()` or similar to check that a container has available checkpoints

Reply:
This is an old comment. I just moved to the right place after the move to cobra. I removed it as what currently happens is to check if the container has a checkpoint.